### PR TITLE
Some minor updates to waveshare-esp32s2-pico board definition

### PR DESCRIPTION
**Pull request by bill88t**
I tested flashing via idf without tinyuf2.

### CODE REVIEW

File: ports/espressif/boards/waveshare_esp32s2_pico/sdkconfig [hunk 1]
```diff
@@ -1,9 +1,11 @@
+#
+# SPI RAM config
+#
 CONFIG_ESP32S2_SPIRAM_SUPPORT=y
-
 CONFIG_DEFAULT_PSRAM_CLK_IO=30
 CONFIG_DEFAULT_PSRAM_CS_IO=26
-
 CONFIG_SPIRAM_TYPE_ESPPSRAM64=y
+CONFIG_SPIRAM_SIZE=8388608
```
Comment (dhalbert) [inline ports/espressif/boards/waveshare_esp32s2_pico/sdkconfig:8]: This page says it is 4MB: https://www.waveshare.com/wiki/ESP32-S2-Pico. But yours is 8MB? Is it clearly a genuine WaveShare board?

Comment (bill88t) [reply ports/espressif/boards/waveshare_esp32s2_pico/sdkconfig:8]: 3rd bulletpoint.

Comment (bill88t) [reply ports/espressif/boards/waveshare_esp32s2_pico/sdkconfig:8]: 8mb psram, 4mb flash.

Comment (dhalbert) [reply ports/espressif/boards/waveshare_esp32s2_pico/sdkconfig:8]: Ah, got it, sorry, thanks.

**Review by dhalbert — APPROVED** [review]

**Pull request merged**